ironic-scripts/runironic-api

```diff
@@ -14,9 +14,6 @@ while true ; do
   sleep 5
 done
 
-
-# TODO: Delete the line of code below and uncomment the next line after the PR https://github.com/metal3-io/baremetal-operator/pull/728 goes in
-[ ! "$(netstat -l | grep ${HTTP_PORT})" ] && . /bin/configure-httpd-ipa.sh  
 # . /bin/configure-httpd-ipa.sh  
```

Comment:
I think the comment suggested uncommenting this line.

Reply:
yes, I forgot to remove the comment......